schema_utils.py

```diff
@@ -155,13 +155,13 @@ def normalize_against_schema(
         assert isinstance(obj, int), ('Expected int, received %s' % obj)
         normalized_obj = obj
     elif schema[SCHEMA_KEY_TYPE] == SCHEMA_TYPE_HTML:
-        assert isinstance(obj, python_utils.BASESTRING), (
+        assert isinstance(obj, (str, bytes)), (
```

Comment:
Here it makes sense to use both types because of the way normalize_against_schema is tested. It needs to raise an exception at that point.
Compairing agaist just one type would mean to remove that assert altogether and thus changing the way its tested too.

what do you think @vojtechjelinek